hw/ip/otp_ctrl/data/otp_ctrl.hjson.tpl

```diff
@@ -1032,8 +1026,267 @@
       }
     ],
 
-    // these CSRs are defined in a separate hjson specific to the closed source OTP wrapper.
+    // OTP wrapper-specific registers
     prim: [
+      { name: "CSR0",
```

Comment:
Note: there is a corresponding PR in the foundry repo that shows the closed-source Hjson couterpart for this change with the alias register definitions.